app/decorators/user_decorator.rb

```diff
@@ -40,7 +40,7 @@ def masked_two_factor_phone_number
   end
 
   def identity_not_verified?
-    user.active_profile ? false : true
+    !user.verified?
```

Comment:
I'm probably not Ruby enough either, since I also prefer the latter.

However, I'd like to leave it as-is until I hear from someone who *is* Ruby enough. @monfresh @amoose ?